src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,12 +8,17 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String EXAMPLE = "123, Clementi Ave 3, #01-23, 231534";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses MUST be in this format: BLOCK, " +
```

Comment:
Why divide the "block"  and "street, unit and postal code"

Reply:
I divided the string to 2 separate lines to follow the Java Coding Standard (https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html), under "Layout", (2) "Keep lines no longer than 120 chars."

I hope that answers your question!